packages/provider-elasticsearch/src/example-types/filters/step.test.js

```diff
@@ -0,0 +1,44 @@
+import step from './step.js'
+
+const { hasValue, filter, buildQuery } = step
+
+describe('step', () => {
+  const input = {
+    type: 'step',
+    field: 'test',
+    range: [0, 500],
+    steps: [0, 500, 1000],
+  }
+
+  it('checks if range is present and not empty', () => {
+    let withoutinput = { type: 'step', field: 'test' }
```

Comment:
💎 very minor tweak for camelCase
```suggestion
    let withoutInput = { type: 'step', field: 'test' }
```